helfomat-api/src/main/kotlin/de/helfenkannjeder/helfomat/api/organization/OrganizationAssembler.kt

```diff
@@ -0,0 +1,234 @@
+package de.helfenkannjeder.helfomat.api.organization
+
+import com.google.common.base.Objects
+import de.helfenkannjeder.helfomat.api.question.AnsweredQuestionDto
+import de.helfenkannjeder.helfomat.core.organization.*
+import de.helfenkannjeder.helfomat.core.picture.PictureId
+import de.helfenkannjeder.helfomat.core.question.Question
+import de.helfenkannjeder.helfomat.core.question.QuestionId
+
+/**
+ * @author Valentin Zickner
+ */
+object OrganizationAssembler {
+
+    fun toScoredOrganizationDtos(organizations: List<ScoredOrganization>) = organizations.map { toOrganizationDto(it) }
+
+    private fun toOrganizationDto(scoredOrganization: ScoredOrganization): OrganizationDto {
```

Comment:
Make this an extension function of ScoredOrganization like this:

```
fun ScoredOrganization.toOrganizationDto(): OrganizationDto {

}
```

Same applies for all the other methods

Reply:
And move the extension functions to top level, instead of in the Object